src/azure-cli/azure/cli/command_modules/serviceconnector/tests/latest/test_local_connection_scenario.py

```diff
@@ -0,0 +1,1199 @@
+# --------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+
+import unittest
+from azure.cli.core.commands.client_factory import get_subscription_id
+from azure.cli.testsdk import (
+    ScenarioTest,
+    record_only,
+    live_only
+)
+from azure.cli.command_modules.serviceconnector._resource_config import (
+    RESOURCE,
+    SOURCE_RESOURCES,
+    TARGET_RESOURCES
+)
+from azure.cli.testsdk.preparers import ResourceGroupPreparer
+from ._test_utils import CredentialReplacer
+resource_group = 'servicelinker-cli-test-group'
+
+
+@unittest.skip('Test with user account signed in')
+class LocalConnectionScenarioTest(ScenarioTest):
```

Comment:
some commands are not tested, `list-support-types`, `preview-configruations`

Reply:
Added